pandas/tests/plotting/test_datetimelike.py

```diff
@@ -331,7 +332,7 @@ def test_freq_with_no_period_alias(self):
         bts = tm.makeTimeSeries(5).asfreq(freq)
         _, ax = self.plt.subplots()
         bts.plot(ax=ax)
-        assert ax.get_lines()[0].get_xydata()[0, 0] == bts.index[0].toordinal()
```

Comment:
This test, and some similar, are invalid now. The relationship between the plotted xy data and ordinals isn't necessarily stable / valuable.